src/main/java/uk/gov/hmcts/probate/model/Event.java

```diff
@@ -0,0 +1,6 @@
+package uk.gov.hmcts.probate.model;
+
+public enum Event {
```

Comment:
is this enum really State, not Event? Also, maybe we need to refactor StateChangeService class for states to be included like these?

Reply:
It is Event.